clang/lib/CIR/Dialect/Transforms/TargetLowering/LowerFunction.cpp

```diff
@@ -517,6 +555,15 @@ LogicalResult LowerFunction::generateCode(FuncOp oldFn, FuncOp newFn,
   Block *srcBlock = &oldFn.getBody().front();
   Block *dstBlock = &newFn.getBody().front();
 
+  // Ensure both blocks have the same number of arguments in order to
+  // safely merge them
+  CIRToCIRArgMapping IRFunctionArgs(LM.getContext(), FnInfo, true);
+  if (IRFunctionArgs.hasSRetArg()) {
+    auto dst_index = IRFunctionArgs.getSRetArgNo();
+    auto ret_arg = dstBlock->getArguments()[dst_index];
```

Comment:
Oops, is the snake_case here intentional? Same for `RV_addr` above.

Reply:
>  is the snake_case here intentional? 

sure it is) old guys like me don't like camelCase) fixed!)